app/controllers/katello/api/registry/registry_proxies_controller.rb

```diff
@@ -147,8 +148,15 @@ def token
         end
       end
 
+      expiration_seconds = (personal_token.expires_at.to_time - Time.now).seconds.to_int
+      issue_time = Time.now.rfc3339
+
       response.headers['Docker-Distribution-API-Version'] = 'registry/2.0'
-      render json: { token: personal_token.token, expires_at: personal_token.expires_at, issued_at: personal_token.created_at }
+      render json: {
+        token: personal_token.token,
+        expires_in: expiration_seconds,
+        issued_at: issue_time
```

Comment:
Could `personal_token.created_at` be used here and converted into rfc3339 time?

Reply:
Of course. expires_in calculations would need to be changed around a little. To be honest I don't understand why I went with this approach anyways.